apiary.apib

```diff
@@ -3247,7 +3249,8 @@ Create a new project.
     + Attributes (object)
         + title: `New company website` (string, required)
         + description (string, optional)
-        + starts_on: `2016-02-04` (string, required)
+        + starts_on: `2020-06-15` (string, required)
+        + due_on: `2020-09-15` (string, optional)
```

Comment:
Q:  should we mention that the `due_on` date will default to the last phase's due date it this value is omitted?

Reply:
Ok, I documented this.